modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,119 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="journald-receiver_{context}"]
+=== Journald receiver
+
+The Journald receiver is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
```

Comment:
We just lower-case journald in the docs. 

```suggestion
The journald receiver is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
```

Reply:
Thank you. I'm actually capitalizing all the component names now, so no need to worry about this one.

Reply:
However, there is one more instance that I am lowercasing. Thank you again.